lib/team.rb

```diff
@@ -0,0 +1,28 @@
+require 'pry'
+class Team
+  attr_reader :country,
+              :players
+
+  attr_accessor :eliminated
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def add_player(player)
+    @players << player
+  end
+
+  def players_by_position(inquired_position)
```

Comment:
This definitely works, but with `#find_all` you do not need to store and return the new array - it does that for you!  You could refactor to something like this:

```ruby
def players_by_position(inquired_position)
  @players.find_all do |player|
    player.position == inquired_position
  end
end
```